batch/test/test_batch.py

```diff
@@ -674,7 +674,7 @@ def test_create_idempotence(client: BatchClient):
     assert b1.id == b2.id
 
 
-def test_batch_create_validation():
+async def test_batch_create_validation():
```

Comment:
Can you make this async? Not sure if we have async tests enabled here.

Reply:
I think it's on by default. It ran and passed [here](https://ci.hail.is/batches/7002970/jobs/116)